### DISCUSSION TITLE
Filter resources before generating patients

### DISCUSSION THREAD

**Discussion by willdalh** [synthetichealth/synthea#1242] [Q&A]
Hi!

The change from #1178 is great, and makes it easier to manage the generated patients when irrelevant resource types are not exported. However, I would like to see a way of filtering out some resources during generation, mainly to reduce execution time. 

I understand that this should only work with resources that are not dependencies of other resource types. In my case, I would like to avoid generating `Claim`, `ExplanationOfBenefit` and `Coverage`.

Is there any way to achieve this at the current state of Synthea without modifying the code? If not, I can open a feature request for this. 

Note: #1114 seems to want the inverse case, specifying inclusion. Rather, I want to specify exclusion.

**Comment by lhs-open** [comment]
Are you asking how to exclude certain record files? The parameter exporter.csv.excluded_files of similar can set a list of output files to exclude.
